packages/dataviews/src/components/dataviews/style.scss

```diff
@@ -60,6 +60,16 @@
 			color: var(--wp-admin-theme-color);
 		}
 	}
+
+	&.dataviews-view-grid__primary-field--clickable span,
+	> .dataviews-view-table__cell-content--clickable span {
+		color: $gray-900;
+
+		&:hover {
+			color: var(--wp-admin-theme-color);
+		}
+		@include link-reset();
+	}
```

Comment:
Why did you add these styles (per curiosity)

Reply:
They were related to all our views. Before we had similar styles in patterns and templates, though they were also coming by using buttons and links in titles. 

Now they are all the same internally (span) and handle the redirect with `onClickItem`, so these styles are used by all views.